drivers/timer/Kconfig.stm32_lptim

```diff
@@ -0,0 +1,44 @@
+# Kconfig - STM32 LPTIM configuration options
+#
+# Copyright (c) 2019 STMicroelectronics
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+config STM32_LPTIM_TIMER
+	bool "STM32 Low Power Timer"
+	depends on (SOC_SERIES_STM32L4X)
```

Comment:
Should it be `(... || SOC_SERIES_STM32WBX)`?

Reply:
I'm not sure about the scope of these changes (L4x only?) but similarly `|| SOC_SERIES_STM32L0X` could be added as well. `LPTIM` is supported by almost any STM32L series MCUs with some exceptions: all of `STM32L1x` range and these six MCUs: `STM32L051C8Ux`, `STM32L052C8Ux`, `STM32L053C8Ux`, `STM32L063C8Ux`, `STM32L073CZUx`, `STM32L552QCUx`.